game/scripts/vscripts/internal/gamemode.lua

```diff
@@ -161,7 +161,7 @@ function GameMode:_CaptureGameMode()
     mode:SetTopBarTeamValuesOverride ( USE_CUSTOM_TOP_BAR_VALUES )
     mode:SetTopBarTeamValuesVisible( TOP_BAR_VISIBLE )
     mode:SetUseCustomHeroLevels ( USE_CUSTOM_HERO_LEVELS )
-    mode:SetCustomHeroMaxLevel ( MAX_LEVEL )
+    --mode:SetCustomHeroMaxLevel ( MAX_LEVEL )
```

Comment:
No we don't, we use the global to generate the table with the needed xp for each level. `SetCustomHeroMaxLevel` is deprecated and not needed when using `SetCustomXPRequiredToReachNextLevel`.